package.json

```
@@ -2,13 +2,19 @@
   "name": "mapeo-schema",
   "version": "2.0.2",
   "description": "JSON schema and flow types for Mapeo",
-  "main": "index.js",
+  "main": "src/index.js",
+  "type": "module",
   "scripts": {
-    "build": "node ./scripts/build.js",
-    "docs": "rimraf docs && mkdirp docs && jsonschema2md -o docs -d schema -e json && rimraf docs/*.json",
-    "prepublish": "npm run build",
-    "test": "npm run build && tape 'test/**.js' && flow check test/valid_flow",
-    "release": "standard-version"
+    "example": "node examples/schema_test.js",
+    "protobuf": "buf generate ./proto",
+    "jsonschema": "scripts/./jsonts",
+    "tsc": "tsc",
+    "tsc-index": "tsc index.js -d --emitDeclarationOnly --allowJs --allowSyntheticDefaultImports --module es2022 --target es2019 --moduleResolution node",
+    "generate": "node scripts/generate.js",
+    "doc": "typedoc --plugin typedoc-plugin-markdown",
+    "clean": "rm -rf types dist docs",
+    "build": "npm-run-all protobuf jsonschema generate tsc tsc-index doc",
+    "test": "tape 'test/**/*.js'"
```

```suggestion
    "prepare": "npm run build",
    "test": "tape 'test/**/*.js'"
```

This will make it so the schemas get built on `npm install`. I think in the future we could do this step (and `npm publish`) in a github actions job, but this works for now.